sematic/ui/packages/common/src/pages/GettingStarted/index.tsx

```diff
@@ -0,0 +1,173 @@
+import styled from "@emotion/styled";
+import theme from "src/theme/new";
+import Typography from "@mui/material/Typography";
+import { ShellCommandRelaxed } from "src/component/ShellCommand";
+import { useTheme } from "@mui/material";
+import { SiDiscord, SiGithub, SiLinkedin, SiYoutube, SiTwitter } from "react-icons/si";
+import { MdMenuBook } from "react-icons/md";
+
+export const Container = styled.div`
+    width: 100%;
+    height: 100%;
+    display: flex;
+
+    h1 {
+        margin-block-start: 0;
+        margin-block-end: ${theme.spacing(5)};
+    }
+
+    p {
+        margin-block-start: 0;
+        margin-block-end: ${theme.spacing(5)};
+        font-size: ${theme.typography.fontSize}px;
+    }
+
+    a {
+        color: ${theme.palette.black.main};
+
+        &:hover {
+            color: ${theme.palette.primary.main};
+        }
+    }
+`;
+
+const common = `
+    padding: ${theme.spacing(10)} ${theme.spacing(5)};
+`
+
+const Left = styled.div`
+    width: 50%;
+    border-right: 1px solid ${theme.palette.p3border.main};
+
+    ${common}
+`;
+
+const Right = styled.div`
+    width: 50%;
+    overflow-y: auto;
+    scrollbar-gutter: stable;
+
+    ${common}
+`;
+
+const StyledShellCommand = styled(ShellCommandRelaxed)`
+    margin-bottom: ${theme.spacing(5)};
+`;
+
+const StyledHeadline = styled.h1`
+    height: 50px;
+    margin-block-end: 0!important;
+    display: flex;
+    align-items: center;
+`;
+
+const LinkRow = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-start;
+    height: 50px;
+
+    a {
+        display: flex;
+        text-decoration: none;
+    }
+
+    span {
+        display: flex;
+        align-items: center;
+        font-weight: ${theme.typography.fontWeightBold};
+
+        &:first-of-type {
+            margin-right: ${theme.spacing(3)};
+        }
+    }   
+`;
+
+const GettingStarted = () => {
+    const theme = useTheme();
+
+    return (
+        <Container>
+            <Left>
+                <h1>Set your API key</h1>
+                <p>To get started, store your API key in your local settings files located at ~/.sematic/settings.</p>
```

Comment:
The name of the file is incorrect. Please also apply formatting:
```suggestion
                <p>To get started, store your API key in your local settings file located at <code>~/.sematic/settings.yaml</code>.</p>
```